### PR TITLE
Clean up some duplicated code in cmd/

### PR DESCRIPTION
**Pull request by hawkowl**
### Which issue this PR addresses:

Part of [issues.redhat.com/browse/ARO-4895](https://issues.redhat.com/browse/ARO-4895) (breaking out #3571)

### What this PR does / why we need it:

Reduces some duplication in cmd/ that will get repeated in MIMO

### Test plan for issue:

Unit tests, E2E

### Is there any documentation that needs to be updated for this PR?
N/A

### How do you know this will function as expected in production? 

E2E

### CODE REVIEW

File: pkg/util/service/helpers.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+func DBName(isLocalDevelopmentMode bool) (string, error) {
```
[the remark this answers is not in the document]
Comment (jaitaiwan) [reply pkg/util/service/helpers.go:13]: Agree with this one

File: pkg/util/service/database.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package service
```
[the remark this answers is not in the document]
Comment (mociarain) [reply pkg/util/service/database.go:1]: I agree with Ayato. Maybe splitting them into `util/database` and `util/encryption` would help.

What do you think?

**Review by mociarain — COMMENTED** [review]
Small suggestion but this is a great change imo

**Review by jaitaiwan — APPROVED** [review]
LGTM

**Pull request merged**